source/atlas-cli-deploy-local.txt

```diff
@@ -409,6 +409,16 @@ atlas deployments respectively.
       #. Specify how you want to connect to the deployment and press 
          :kbd:`Enter`..
 
+         You can connect with any of the following or retrieve the
+         connection string: 
+
+         - {+mongosh+} if you installed :binary:`~bin.mongosh`
+         - MongoDB Compass if you installed :compass:`Compass </>`
+         - Visual Studio Code if you installed `Visual Studio Code <https://code.visualstudio.com/download>`__,
+           `Visual Studio Code CLI
+           <https://code.visualstudio.com/download>`__, and `MongoDB for
+           VS Code Extension <https://www.mongodb.com/docs/mongodb-vscode/install/>`__
```

Comment:
Nit - I think we can just list the clients and link to the installation page to simplify

```suggestion
         - {+mongosh+} 
         - :compass:`MongoDB Compass </>`
         - Visual Studio Code through the `MongoDB for VS Code Extension <https://www.mongodb.com/docs/mongodb-vscode/install/>`__
```